Tests/scripts/infrastructure_tests/test_collect_tests.py

```diff
@@ -632,7 +634,7 @@ def test_number_of_file_types():
 
         - Removed type:    Decrease the number here.
     """
-    assert len(FileType) == 76
+    assert len(FileType) == 77
```

Comment:
have you added it to `IGNORED_FILE_TYPES`?

Reply:
https://github.com/demisto/content/pull/29667/files#diff-2a6c9234e44f974b6213005e4fcff734db5dd00c16d48b377859d606b215ea81R117